server/loginserver.ts

```diff
@@ -216,22 +215,29 @@ class LoginServerInstance {
 		this.requestTimerPoke();
 	}
 	getLog() {
-		return this.requestLog + (this.lastRequest ? ' (' + Chat.toDurationString(Date.now() - this.lastRequest) + ' since last request)' : '');
+		return `${this.requestLog}` +
+		`${this.lastRequest ? ` (${Chat.toDurationString(Date.now() - this.lastRequest)} since last request)` : ''}`;
 	}
 }
 
-let LoginServer = Object.assign(new LoginServerInstance(), {
+const LoginServer = Object.assign(new LoginServerInstance(), {
 	TimeoutError,
 
 	ladderupdateServer: new LoginServerInstance(),
 	prepreplayServer: new LoginServerInstance(),
 });
 
 FS('./config/custom.css').onModify(() => {
-	LoginServer.request('invalidatecss');
+	LoginServer.request('invalidatecss').catch((err: string) => {
```

Comment:
id move both these calls into a method where you then then handle the error in one place.

Reply:
Though I'm also guessing the 'handling' will just be to `// tslint:disable`` and ignore the possibility for errors - I dont know where `console.err` logs go on the server or whether anyone looks at them, and I dont know how much we care if the 'invalidatecss' require fails

Reply:
@Zarel

Reply:
We very much don't want to `console.err`. This should definitely just be `tslint:disable`d. Uncaught promises will go directly to Crashlogger which is exactly where we want them.

Reply:
did